flag_string.go

```diff
@@ -57,7 +57,8 @@ func (f *StringFlag) GetValue() string {
 
 // Apply populates the flag given the flag set and environment
 func (f *StringFlag) Apply(set *flag.FlagSet) error {
-	if val, ok := flagFromEnvOrFile(f.EnvVars, f.FilePath); ok {
+	// TODO: how to report source?
```

Comment:
Question: Report source in what situation? Like the above, if there aren't any errors here, I don't think we have anything to report.

Reply:
Removed.